doc/source/analyzing/objects.rst

```diff
@@ -214,6 +213,34 @@ domain but centered at 0.5 in code units, you can do:::
 This `frb` object then can be queried like a normal fixed resolution buffer,
 and it will return arrays of shape (1024, 1024).
 
+Making Rays
+^^^^^^^^^^^
+
+The slicing syntax can also be used select 1D rays of points, whether along 
+an axis or off-axis. To create a ray along an axis:::
+
+    ortho_ray = ds.r[(500.0, "kpc"), (200, "kpc"):(300.0, "kpc"), (-2.0, "Mpc")]
+
+To create a ray off-axis, use a single slice between the start and end points
+of the ray:::
+
+    start = ((500.0, "kpc"), (0.2, "Mpc"), (100.0, "kpc"))
+    end = ((1.0, "Mpc"), (300.0, "kpc"), (0.0, "kpc"))
```

Comment:
While this might work, I think mixing units like this in the docs is confusing. Why not just say:

```
from yt.units import kpc
start = [500, 200, 100]*kpc
end = [1000, 300, 0]*kpc
```

Or some other variation on that?